docs/usage/change-tracking.md

```diff
@@ -53,9 +53,21 @@ In addition to marking text as inserted or deleted, change tracking can also be
 ```ts
 import { Document } from "docx";
 
-const doc = new Document({
-    features: {
-        trackRevisions: true,
-    },
-});
+const doc = new Document({});
+doc.Settings.addTrackRevisions()
```

Comment:
The old API is much prefered over this one

Can this be reverted?

Reply:
Sure, just reverted this change.